index.html

```diff
@@ -143,7 +143,7 @@
             </div>
           </div>
           <div class="row">
-            <div class="col-xs-6">
+            <div class="col-xs-12 col-md-12">
```

Comment:
This will cause Org card widths on desktop to be full width. Shouldn't this, and the below be col-xs-12 col-md-6 ?